app/javascript/redux/greetings/greetings.js

```diff
@@ -0,0 +1,25 @@
+import store from "../configureStore"
+
+export const DISPLAY_GREETING = 'helloRailsReact/greetings/DISPLAY_GREETING'
+
+const initialState = []
+
+export const displayGreeting = () => {
+  fetch(`api/v1/greeting.json`)
+    .then((response) => response.json())
+    .then((json) => store.dispatch({
+      type: DISPLAY_GREETING,
+      payload: json
+    }))
+}
```

Comment:
**Changes Required**
- [ ] Let us follow the `SOLID` principle here. The `Action` should be in it's own file. This will enable us to follow single responsibility for each file.